src/components/Header/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Button, Grid, Typography } from '@mui/material';
+import { useNavigate } from 'react-router-dom';
+import NavBar from '../NavBar';
+
+const Header = () => {
+  const navigate = useNavigate();
+  const { council } = useSelector((state) => state.siteCondition);
+
+  const headerLogo = () => {
```

Comment:
I think this would be better done with a switch statement passing in the council value and if the value doesn't match any of the 4 councils (cases) the default will be the PSA logo. The Header.js file on the selector line 105 has an example of this.